### DISCUSSION TITLE
[FEATURE REQ] Add to top/bottom of queue in transmission

### DISCUSSION THREAD

**Discussion by kattjevfel** [autobrr/autobrr#1691] [Ideas]
**Is your feature request related to a problem? Please describe.**
The queue might be full and have many slow torrents, but I want the new shiny torrent _now_

**Describe the solution you'd like**
An option under Instance --> Rules similar to qbittorrent to select priority.

**Describe alternatives you've considered**
Manually changing the queue position afterwards, but that half defeats the purpose of an automation tool.

**Additional context**
Option as seen with qbittorrent:
![2024-06-03_20-29-14](https://github.com/autobrr/autobrr/assets/6226312/6992bede-aa62-4cc6-9a3e-bebad894e989)

**Comment by blixten85** [comment]
I would like this when using the *arrs as the client too
